integration/fixtures/file/directory.toml

```diff
@@ -0,0 +1,10 @@
+defaultEntryPoints = ["http"]
+
+[entryPoints]
+  [entryPoints.http]
+  address = ":8000"
+
+logLevel = "DEBUG"
```

Comment:
The `logLevel` should to be defined before the `entrypoints` otherwise the logger will not work correctly.